cmake/GSSAPI.cmake

```diff
@@ -0,0 +1,14 @@
+
+function(find_gssapi_library)
+
+  find_library(GSSAPI_LIBRARIES NAMES libgssapi.so.3 libgssapi.so gssapi gssapi )
```

Comment:
Looks like this is missing `libgssapi_krb5.so` from #1157

Reply:
Good point let me pull in the changes from Dev.